doc/client_lua_api.md

```diff
@@ -841,17 +841,25 @@ Please do not try to access the reference until the camera is initialized, other
 * `get_camera_mode()`
     * Returns with same syntax as above
 * `get_fov()`
+	* All values are in degrees
     * Returns:
 
 ```lua
      {
-         x = number,
-         y = number,
-         max = number,
-         actual = number
+         x = float,        -- Horizontal FOV
+         y = float,        -- Vertical FOV
+         max = float,      -- The maximum of x and y
+         actual = integer, -- What the FOV of the player would be (not adjusted for zooming)
+         zoom = integer    -- What the FOV of the player when zooming would be
```

Comment:
What's the meaning of 'would be' in these lines? How is it different from x,y? This needs clarifying.